docs/developer_guide.md

```diff
@@ -1,13 +1,13 @@
 # Developer guide
 
 ## Prerequisites
-- docker
+- Docker
 - opm https://github.com/operator-framework/operator-registry/releases
 - operator-sdk https://github.com/operator-framework/operator-sdk/releases
+- olm https://github.com/operator-framework/operator-lifecycle-manager/releases
 - golang 1.19
 - make
-- kubectl
-- ubuntu20 or redhat8 OS
+- Ubuntu20 or RedHat8 OS
```

Comment:
```suggestion
- Ubuntu22 or RedHat8 OS
```

Reply:
For now, this has only been tested on Ubuntu20

Reply:
update: it also works on Ubuntu 22